yml/apdb.yaml

```
@@ -1558,6 +1544,11 @@ tables:
     description: Covariance between x and y.
     fits:tunit: pixel**2
     ivoa:ucd: stat.covariance
+  - name: centroid_flag
+    "@id": "#DiaSource.centroid_flag"
+    datatype: boolean
+    description: General failure flag, set if anything went wrong
```

Ugh, I don't like this. I'm trying to figure out where this one comes from deeper in the code and not having luck. We can at least make it better here, though:

```suggestion
    description: General centroid algorithm failure flag; set if anything went wrong when fitting the centroid. Another centroid flag field should also be set to provide more information.
```